pandas/tests/reductions/test_reductions.py

```diff
@@ -960,6 +960,18 @@ def test_min_max(self):
         assert np.isnan(_min)
         assert _max == 1
 
+        cat = Series(Categorical(
+            ["a", "b", "c", "a"], categories=["b", "c", "d"], ordered=True))
+        _min = cat.min(numeric_only=True)
+        _max = cat.max(numeric_only=True)
+        assert _min == "b"
+        assert _max == "c"
+
+        _min = cat.min(numeric_only=False)
+        _max = cat.max(numeric_only=False)
+        assert np.isnan(_min)
+        assert _max == "c"
```

Comment:
Can you also add the case without specifying `numeri_only`? (so to make sure we test the default, to be equal to `numeric_only=False` at this point)

Reply:
Those are already present above the newly added tests!

Reply:
But not for the case with missing values?

Reply:
There is both a case with all values present and one with explicit NaNs, right?

Reply:
Ah, yes, sorry, didn't look at the existing tests!